### ISSUE TITLE
Add profession metadata to the MP database

### ISSUE THREAD

**Issue by fredrik1984** [swerik-project/riksdagen-persons#26]
Profession is a metadata category that would be good to eventually add to the MP database.

Agustin Goenaga (Lund University) and his project colleagues have added profession for MPs in the 19th century within their project: https://portal.research.lu.se/sv/projects/the-politics-of-state-building-studying-investments-in-state-capa 

They have used the Swerik MP database and we are welcome to integrate their profession metadata into our MP database. We have got two CSV files, attached here, with this metadata. Agustin gave the following description of these two files:

1. “Speakers_19th.csv” includes the “raw” profession as coded by our RA based mainly on the biographical dictionaries and Wikipedia.
2. “mp_metadata_districtcodes.csv” includes the same profession variable, as well as additional dummy variables that we created in which we tried to systematize some of the professions into more general occupational groups. MPs may have values of 1 in multiple occupational groups if they were listed as having multiple “professions” in the dictionaries. I’m not sure how systematic the RA was when coding more than one occupation, since her instructions were to code the first or more prominent one, but at least we can trace back to the original professions in the “raw” variable. The speaker_id variable is the same as your wiki_id.

Hence, we should integrate Agustin's MP profession metadata into Swerik's MP database.

[mp_metadata_districtcodes[95].csv](https://github.com/user-attachments/files/17485327/mp_metadata_districtcodes.95.csv)
[speakers_19th[37].csv](https://github.com/user-attachments/files/17485328/speakers_19th.37.csv)

**Comment by MansMeg** [comment]
I think we should discuss what should be included in the database. I think profession is borderline since it doesnt connect to mps role. 

I think we can help adding it to wikidata instead. Then people can use it by connecting to wikidata.

**Comment by fredrik1984** [comment]
That is true, having it on Wikidata is more suitable.

**Comment by BobBorges** [comment]
What level of urgency are we talking about here? 

Assuming each profession already has a Q-number on wikidata, it would be straightforward to add the professions in the speakers_19th... file to wikidata. However (!) all claims added to wikidata should have a source -- there is no source indicated in the csv file. 

Further, adding to wikidata is one thing, but if we do it (and we have source information), we should also police those entries in the same way we do with other parts of our data set -- have a `profession.csv` file with columns `["person_id", "start", "end", "profession"]`  and test that our additions are not deleted/overwritten with every pull.

I agree with @MansMeg that this is a borderline case, whether or not to include this variable, however, I think it _is_ relevant to the overall set of potential use-cases for the corpus -- i.e. the professions of parliamentarians (just glancing at the sheet) are not proportionally representative (I would guess) of all professions in the country; why? how does that affect political discourse? 

_If_ there are reasonable sources for the professions data, I think it would be a useful variable to introduce to Swerik, via wikidata and the other workflows that we use.

**Comment by fredrik1984** [comment]
I don't see this as an urgent issue. Mostly because of the borderline case of this metadata category. I think that the ideal case would be if a researcher would be interested in adding this information to Wikidata for their own purpose. 

I think Agustin told me that they used the bio books to get the profession data. But that needs to be confirmed, 

Maybe we should just keep this issue, with the CSV files in it, in the backlog. We have more important stuff to do at the moment (and most likely in the future too).

**Comment by salgo60** [comment]
FYI: I did a check  Aug 14, 2021 of the status of professions in Alvin, Arken, Historiska museet, Kungliga biblioteket - LIBRISXL, Levande musikarv, Litteraturbanken , Svenskt Kvinnobiografiskt lexikon, SBL see GITHUB[salgo60/HISCOKoder](https://github.com/salgo60/HISCOKoder) I also tried to get more people to care about occupations see #1

The major problem I feel is that we **dont see many sources using  [HISCO codes - history of works](https://iisg.amsterdam/en/data/data-websites/history-of-work)** its just text strings without good definitions...

**Sources that has HISCOcodes**

* Mats Hayen has a project ["Tidigmoderna konkurser!"](http://www.tidigmodernakonkurser.se/) som har HISCOkoder se ["Titelklassificering"](http://www.tidigmodernakonkurser.se/index.php/page/3)
[Titelklassificering](http://www.tidigmodernakonkurser.se/index.php/page/3) / [Undergrupp i Hisco](http://www.tidigmodernakonkurser.se/index.php/extended/subject)
* Swepop - [swedpop.se](https://swedpop.se/)...
[HISCO codes and description](https://swedpop.se/wp-content/uploads/2021/06/HISCO-codes-and-description.pdf) Version: June 28, 2021
...

Exempel what I checked...

<img width="953" alt="image" src="https://github.com/user-attachments/assets/0c52c3cf-b26e-43da-af04-6cd2cbc53cd4">

**SWERIK connected persons professions in Wikidata** its a mess BUT

* [SPARQL](https://w.wiki/Bj$P) 

<img width="1257" alt="image" src="https://github.com/user-attachments/assets/b5ac151d-c2b2-4279-9a4d-3ab4e6a28059">

<img width="884" alt="image" src="https://github.com/user-attachments/assets/272c791a-8035-45dd-8c7b-ad0064dd6b8a">

**Comment by salgo60** [comment]
### Wikidata profession of father of person connected to SWERIK
* far from complete

<img width="897" alt="image" src="https://github.com/user-attachments/assets/d4564bdc-5e3c-4288-ae6d-79a7d6364fe8">

* [SPARQL](https://w.wiki/BkRr)  

<img width="1389" alt="image" src="https://github.com/user-attachments/assets/7c887aa0-fdc0-48a4-a374-19716a2c28ce">

**Comment by salgo60** [comment]
> That is true, having it on Wikidata is more suitable.

Wikidata is just a POC - proof of concept... use it but quality assure it and add better sources is my take...

We can see that RAÄ have given up and now also use Wikidata see [FB project description](https://www.facebook.com/search/top?q=riksantikvarie%C3%A4mbetet) I can see a **red flag** that they are not on GITHUB....

<img width="957" alt="image" src="https://github.com/user-attachments/assets/9d6c9729-ac9c-41de-abf8-50f01ac68701">

----
Research project that looks cool [Wikidata:WikiProject Reference Verification](https://www.wikidata.org/wiki/Wikidata:WikiProject_Reference_Verification)

* ProVe: [Automated PROvenance VErification of Knowledge Graphs against Textual Sources](https://www.wikidata.org/wiki/Wikidata:ProVe)
  * [GITHUB King-s-Knowledge-Graph-Lab/ProVe](https://github.com/King-s-Knowledge-Graph-Lab/ProVe)

**Comment by MansMeg** [comment]
I think we can close this issue, sicne this can be accessed from wikidata instead? @fredrik1984 

See #118 .

**Comment by fredrik1984** [comment]
Yes, it can be closed. Also because I guess MP profession is not something that is included in parliament's official MP info. At least it has nothing to do with the role as MP, as noted above.

**Comment by MansMeg** [comment]
Great!
